ledger/controllers/grants.js

```diff
@@ -303,6 +341,31 @@ v3.read = {
   }
 }
 
+/*
+  GET /v4/grants
+*/
+
+v4.read = {
+  handler: getGrantV4(4, chooseFromAvailablePromotionsByType),
+  description: 'See if a v4 promotion is available',
+  tags: [ 'api' ],
+
+  validate: {
+    query: {
+      lang: Joi.string().regex(localeRegExp).optional().default('en').description('the l10n language'),
+      paymentId: Joi.string().guid().optional().description('identity of the wallet')
+    }
+  },
```

Comment:
I see we're using a `type` query parameter in the handler function https://github.com/brave-intl/bat-ledger/pull/468/files#diff-7fb1e89ea3f41276b410cb21071d95ddR1164, should we add validations for it?

Reply:
yeah, i should remove this